lib/common/validation.js

```diff
@@ -1252,32 +1355,49 @@ exports.validateUpdateVmParams = function (vmapi, vm, obj, callback) {
 };
 
 function validateListVmsParams(params, callback) {
-    var listVmValidators = {
-        owner_uuid: createValidateUUIDFn('owner_uuid'),
-        server_uuid: createValidateUUIDFn('server_uuid'),
-        uuid: createValidateUUIDFn('uuid'),
-        uuids: createValidateCSVFn('uuids', validUUID),
-        brand: createValidateStringsListFn('brand', VALID_VM_BRANDS),
-        alias: createValidateStringFn('alias', {re: ALIAS_RE}),
-        state: createValidateStringsListFn('state', VALID_VM_STATES),
-        ram: createValidateStringFn('ram', {re: RAM_RE}),
-        predicate: createValidateJSONPredicateFn('predicate'),
-        query: createValidateStringFn('query'),
-        docker: createValidateBooleanFn('docker'),
-        image_uuid: createValidateUUIDFn('image_uuid'),
-        billing_id: createValidateUUIDFn('billing_id'),
-        create_timestamp: createValidateTimestampFn('create_timestamp'),
-        package_name: createValidateStringFn('package_name'),
-        package_version: createValidateStringFn('package_version'),
-        fields: createValidateVmFieldsFn('fields'),
-        tag: createValidateTagFn(),
-        sort: createValidateStringFn('sort'),
-        limit: createValidateNumberFn('limit'),
-        offset: createValidateNumberFn('offset')
-    };
+    async.series([
+        function validateSingleParams(next) {
+            var listVmValidators = {
+                owner_uuid: createValidateUUIDFn('owner_uuid'),
+                server_uuid: createValidateUUIDFn('server_uuid'),
+                uuid: createValidateUUIDFn('uuid'),
+                uuids: createValidateCSVFn('uuids', validUUID),
+                brand: createValidateStringsListFn('brand', VALID_VM_BRANDS),
+                alias: createValidateStringFn('alias', {re: ALIAS_RE}),
+                state: createValidateStringsListFn('state', VALID_VM_STATES),
+                ram: createValidateStringFn('ram', {re: RAM_RE}),
+                predicate: createValidateJSONPredicateFn('predicate'),
+                query: createValidateStringFn('query'),
+                docker: createValidateBooleanFn('docker'),
+                image_uuid: createValidateUUIDFn('image_uuid'),
+                billing_id: createValidateUUIDFn('billing_id'),
+                create_timestamp: createValidateTimestampFn('create_timestamp'),
+                package_name: createValidateStringFn('package_name'),
+                package_version: createValidateStringFn('package_version'),
+                fields: createValidateVmFieldsFn('fields'),
+                tag: createValidateTagFn(),
+                sort: createValidateSortFn('sort', {markerParamName: 'marker'}),
```

Comment:
It seems like we don't actually use the `"markerParamName"` option.  Can we stop passing it in?